app.R

```diff
@@ -999,8 +997,9 @@ ui <- fluidPage(
               HTML("<br><br>"),
               fileInput(
                 "custom_zip", 
-                "Upload Custom Data ZIP File (.zip)",
-                accept = ".zip"
+                "Upload Custom Data Files (.xlsx, .csv)",
+                accept = c(".xlsx", ".csv"),
```

Comment:
do you also need to specify ".xls" for older versions of excel?

Reply:
Since the only excel file is the edited Metadata.xlsx file, no -- the rest of the files must be CSVs.